app/components/decision_component.html.erb

```diff
@@ -11,6 +11,13 @@
         end
       end
 
```

Comment:
We now show the means test on the decision summary card (Crown Court or Magistrates Court)